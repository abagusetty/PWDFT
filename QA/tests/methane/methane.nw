Title "CH4  test"

memory stack 500 mb heap 750 mb global 10 mb

start ch4-pspw

echo

#permanent_dir ./perm
#scratch_dir   ./perm


geometry noautosym noautoz
C        -0.000167     0.000612     0.000143
H         1.074360     0.076132    -0.029375
H        -0.290234    -0.637689     0.818651
H        -0.352445    -0.425876    -0.925066
H        -0.429688     0.980144     0.134092
end


nwpw
   simulation_cell
     SC 26.0
   end
   
   lmbfgs

   xc pbe
   cutoff 30.0
   #1d-slab
end

task pspw optimize


