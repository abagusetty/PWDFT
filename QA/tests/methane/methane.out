pwdft (NWChemEx) - Version 1.0

============================== echo of input deck ==============================
Title "CH4  test"

memory stack 500 mb heap 750 mb global 10 mb

start ch4-pspw

echo

#permanent_dir ./perm
#scratch_dir   ./perm


geometry noautosym noautoz
C        -0.000167     0.000612     0.000143
H         1.074360     0.076132    -0.029375
H        -0.290234    -0.637689     0.818651
H        -0.352445    -0.425876    -0.925066
H        -0.429688     0.980144     0.134092
end


nwpw
   simulation_cell
     SC 26.0
     #boundary_conditions aperiodic
   end
   
   lmbfgs

   xc pbe
   cutoff 30.0
   2d-hcurve
end

task pspw optimize


================================================================================

              NorthwestEx Computational Chemistry Package 1.0.0
           --------------------------------------------------------

                  Pacific Northwest National Laboratory
                           Richland, WA 99354

                         Copyright (c) 2020
                  Pacific Northwest National Laboratory
                       Battelle Memorial Institute

        NWChemEx is an open-source computational chemistry package
                   distributed under the terms of the
                 Educational Community License (ECL) 2.0
        A copy of the license is included with this distribution
                         in the LICENSE.TXT file

                             ACKNOWLEDGMENT
                             --------------

       This software and its documentation were developed at the
       Pacific Northwest National Laboratory, a multiprogram
       national laboratory, operated for the U.S. Department of Energy
       by Battelle under Contract Number DE-AC05-76RL01830. Support
       for this work was provided by the Department of Energy 
       Office of Advanced Scientific Computing and the Office of Basic
       Energy Sciences.

       Job information
       ---------------
       program               = pwdft (NWChemEx)
       build configured      = Sat Jul 08 17:52:31 2023
       source                = /Users/bylaska/Codes/PWDFT/Nwpw
       version               = 1.0
       default psp libraries = /Users/bylaska/Codes/PWDFT/Nwpw/libraryps

       date                  = Sat Jul  8 21:23:20 2023
       nproc                 = 6
       input                 = ch4.nw



First rtdbstr={"constraints":null,"current_task":"task pspw optimize","dbname":"ch4-pspw","driver":null,"foundtask":true,"geometries":{"geometry":{"autosym":0,"autoz":0,"center":1,"charges":[6.0,1.0,1.0,1.0,1.0],"conv":1.88972598858,"coords":[-0.0003161669999037255,0.0011569026975055872,0.0002697102930407356,2.030245430330998,0.1438690093550672,-0.0555112214378637,-0.5484633153293386,-1.205057085539097,1.5470255497536793,-0.6660250588048889,-0.8047885547200013,-1.7481217818750725,-0.8119931633407739,1.8522039797432501,0.2533966167373431],"masses":[12.0,1.008,1.008,1.008,1.008],"nion":5,"symbols":["C","H","H","H","H"],"unita":[1.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,1.0],"velocities":[0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]}},"nwinput_cur":35,"nwinput_lines":["Title \"CH4  test\"","","memory stack 500 mb heap 750 mb global 10 mb","","start ch4-pspw","","echo","","","","","","geometry noautosym noautoz","C        -0.000167     0.000612     0.000143","H         1.074360     0.076132    -0.029375","H        -0.290234    -0.637689     0.818651","H        -0.352445    -0.425876    -0.925066","H        -0.429688     0.980144     0.134092","end","","","nwpw","   simulation_cell","     SC 26.0","     ","   end","   ","   lmbfgs","","   xc pbe","   cutoff 30.0","   2d-hcurve","end","","task pspw optimize","","",""],"nwinput_nlines":38,"nwpw":{"cutoff":[30.0,60.0],"mapping":3,"minimizer":2,"simulation_cell":{"unita":[26.0,0.0,0.0,0.0,26.0,0.0,0.0,0.0,26.0]},"xc":"pbe"},"permanent_dir":".","psp_library_dir":"","scratch_dir":".","title":"CH4  test"}
First task=3


Running geometry optimization calculation - rtdbstr = {"constraints":null,"current_task":"task pspw optimize","dbname":"ch4-pspw","driver":null,"foundtask":true,"geometries":{"geometry":{"autosym":0,"autoz":0,"center":1,"charges":[6.0,1.0,1.0,1.0,1.0],"conv":1.88972598858,"coords":[-0.0003161669999037255,0.0011569026975055872,0.0002697102930407356,2.030245430330998,0.1438690093550672,-0.0555112214378637,-0.5484633153293386,-1.205057085539097,1.5470255497536793,-0.6660250588048889,-0.8047885547200013,-1.7481217818750725,-0.8119931633407739,1.8522039797432501,0.2533966167373431],"masses":[12.0,1.008,1.008,1.008,1.008],"nion":5,"symbols":["C","H","H","H","H"],"unita":[1.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,1.0],"velocities":[0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]}},"nwinput_cur":35,"nwinput_lines":["Title \"CH4  test\"","","memory stack 500 mb heap 750 mb global 10 mb","","start ch4-pspw","","echo","","","","","","geometry noautosym noautoz","C        -0.000167     0.000612     0.000143","H         1.074360     0.076132    -0.029375","H        -0.290234    -0.637689     0.818651","H        -0.352445    -0.425876    -0.925066","H        -0.429688     0.980144     0.134092","end","","","nwpw","   simulation_cell","     SC 26.0","     ","   end","   ","   lmbfgs","","   xc pbe","   cutoff 30.0","   2d-hcurve","end","","task pspw optimize","","",""],"nwinput_nlines":38,"nwpw":{"cutoff":[30.0,60.0],"mapping":3,"minimizer":2,"simulation_cell":{"unita":[26.0,0.0,0.0,0.0,26.0,0.0,0.0,0.0,26.0]},"xc":"pbe"},"permanent_dir":".","psp_library_dir":"","scratch_dir":".","title":"CH4  test"}

          *****************************************************
          *                                                   *
          *             PWDFT PSPW GeoVib Calculation         *
          *                                                   *
          *  [ (Grassmann/Stiefel manifold implementation) ]  *
          *  [              C++ implementation             ]  *
          *                                                   *
          *              version #7.00   02/27/21             *
          *                                                   *
          *    This code was developed by Eric J. Bylaska,    *
          *    Abhishek Bagusetty, David H. Bross, ...        *
          *                                                   *
          *****************************************************
          >>> job started at       Sat Jul  8 21:23:20 2023 <<<

 psp_library: /Users/bylaska/Codes/PWDFT/Nwpw/libraryps



 reading formatted psp filename: ./C.vpp

 reading formatted psp filename: ./H.vpp
 input psi exists, reading from file: ./ch4-pspw.movecs

     ===================  summary of input  =======================

 input psi filename: ./ch4-pspw.movecs

 number of processors used: 6
 processor grid           : 6 x1
 parallel mapping         : 2d-hcurve
 parallel mapping         : balanced

 options:
   boundary conditions  = periodic
   electron spin        = restricted
   exchange-correlation = PBE96 (White and Bird) parameterization

 elements involved in the cluster:
      1: C   valence charge =  4.0  lmax =2
             comment = Parameterized (J.Phys.Chem., vol 100, page 6966) Hamman psp 
             pseudopotential type            =  0
             highest angular component       =  2
             local potential used            =  2
             number of non-local projections =  4
             cutoff =    0.800   0.850   0.850
      2: H   valence charge =  1.0  lmax =1
             comment = Parameterized (Chem.Phys.Lett., vol 322, page 447) Hamman psp 
             pseudopotential type            =  0
             highest angular component       =  1
             local potential used            =  1
             number of non-local projections =  1
             cutoff =    0.800   0.800

 total charge =   0.000

 atom composition:
   C : 1   H : 4

 initial ion positions (au):
    1 C  (   -0.00032    0.00116    0.00027 ) - atomic mass = 12.000
    2 H  (    2.03025    0.14387   -0.05551 ) - atomic mass =  1.008
    3 H  (   -0.54846   -1.20506    1.54703 ) - atomic mass =  1.008
    4 H  (   -0.66603   -0.80479   -1.74812 ) - atomic mass =  1.008
    5 H  (   -0.81199    1.85220    0.25340 ) - atomic mass =  1.008
    G.C. (    0.00069   -0.00252   -0.00059 )
  C.O.M. (   -0.00000   -0.00000   -0.00000 )


 number of electrons: spin up =     4 (   4 per task) down =     4 (   4 per task)

 supercell:
      volume =   17576.00
      lattice:    a1 = <   26.000    0.000    0.000 >
                  a2 = <    0.000   26.000    0.000 >
                  a3 = <    0.000    0.000   26.000 >
      reciprocal: b1 = <    0.242    0.000    0.000 >
                  b2 = <    0.000    0.242    0.000 >
                  b3 = <    0.000    0.000    0.242 >
      lattice:    a =      26.000 b =     26.000 c =      26.000
                  alpha =  90.000 beta =  90.000 gamma =  90.000
      density cutoff = 60.000 fft =  96 x   96 x   96  (  195216 waves    32536 per task)
      wavefnc cutoff = 30.000 fft =  96 x   96 x   96  (   69013 waves    11502 per task)

 Ewald parameters:
      energy cutoff =  60.000 fft =  96 x   96 x   96  (  195216 waves    32536 per task)
      Ewald summation: cut radius =   8.276 and   1
                       Mandelung Wigner-Seitz =  1.76011888 (alpha =  2.83729748 rs = 16.12911276)

 technical parameters:
      fixed step: time step =        5.80  ficticious mass =   400000.00
      tolerance =   1.000e-07 (energy)    1.000e-07 (density)    1.000e-04 (ion)
      max iterations =       1000 (   10 inner   100 outer)
      minimizer = Grassmann lmbfgs



 -----------------------------------------------------------------------------------
 ----------------------------- Geometry Optimization -------------------------------
 -----------------------------------------------------------------------------------

 Optimization parameters:     30 (maxiter)    4.500e-04 (gmax)    3.000e-04 (grms)    1.200e-03 (xrms)    1.800e-03 (xmax)
    maximum number of steps       (maxiter) =   30
    maximum gradient threshold       (Gmax) = 4.500000e-04
    rms gradient threshold           (Grms) = 3.000000e-04
    rms cartesian step threshold     (Xrms) = 1.200000e-03
    maximum cartesian step threshold (Xmax) = 1.800000e-03

    fixed trust radius              (Trust) = 3.000000e-01
    number lmbfgs histories   (lmbfgs_size) =    9


 -----------------------------------------------------------------------------------
 -----------------------------    Initial Geometry     -----------------------------
 -----------------------------------------------------------------------------------

 ---------------------------------
         Initial Geometry         
 ---------------------------------

 == Bonds, Angles, and Torsions ==

 Number of Atoms = 5


 Units are Angstrom for bonds and degrees for angles

      Type          I     J     K     L     M      Value
      ----------- ----- ----- ----- ----- ----- ----------
    1 Stretch        C1    H2                      1.07758
    2 Stretch        C1    H3                      1.07774
    3 Stretch        C1    H4                      1.07796
    4 Stretch        C1    H5                      1.07792
    5 Bend           H2    C1    H3              109.31078
    6 Bend           H2    C1    H4              109.27424
    7 Bend           H2    C1    H5              109.69786
    8 Bend           H3    C1    H4              109.24251
    9 Bend           H3    C1    H5              109.66849
   10 Bend           H4    C1    H5              109.63130


 == XYZ format geometry ==

5

C        -0.000167     0.000612     0.000143
H         1.074359     0.076132    -0.029375
H        -0.290234    -0.637688     0.818650
H        -0.352445    -0.425876    -0.925066
H        -0.429688     0.980144     0.134092



 ---------------------------------
     Calculate Initial Energy     
 ---------------------------------

     ================= Grassmann lmbfgs iteration =================
          >>> iteration started at Sat Jul  8 21:23:20 2023  <<<
     iter.                   Energy          DeltaE        DeltaRho
     --------------------------------------------------------------
        10      -8.107103142763e+00   -7.757457e-08    3.952313e-09
     *** tolerance ok. iteration terminated
          >>> iteration ended at   Sat Jul  8 21:23:21 2023  <<<

     =============  energy results (Molecule object)  =============


 number of electrons: spin up=     4.00000  down=     4.00000 (real space)


 total     energy    :   -8.1071031428e+00 (   -1.62142e+00 /ion)
 total orbital energy:   -3.3357292338e+00 (   -8.33932e-01 /electron)
 hartree energy      :    1.1896121199e+01 (    2.97403e+00 /electron)
 exc-corr energy     :   -3.1972599286e+00 (   -7.99315e-01 /electron)
 ion-ion energy      :    6.1834259589e+00 (    1.23669e+00 /ion)

 kinetic (planewave) :    6.3718372500e+00 (    1.59296e+00 /electron)
 V_local (planewave) :   -2.8602959092e+01 (   -7.15074e+00 /electron)
 V_nl    (planewave) :   -7.5826852986e-01 (   -1.89567e-01 /electron)
 V_Coul  (planewave) :    2.3792242398e+01 (    5.94806e+00 /electron)
 V_xc    (planewave) :   -4.1385812599e+00 (   -1.03465e+00 /electron)
 Viral Coefficient   :   -1.5235113678e+00

 orbital energy:
    -3.4787485e-01 (  -9.466eV)
    -3.4805391e-01 (  -9.471eV)
    -3.4888153e-01 (  -9.494eV)
    -6.2305433e-01 ( -16.954eV)

 == Center of Charge ==

 spin up    = (    0.0003    -0.0010    -0.0003 )
 spin down  = (    0.0003    -0.0010    -0.0003 )
      total = (    0.0003    -0.0010    -0.0003 )
 ionic      = (    0.0003    -0.0011    -0.0003 )

 == Molecular Dipole wrt Center of Mass ==

 mu   = (    0.0004    -0.0008    -0.0001 ) au
 |mu| =      0.0009 au (     0.0023 Debye )


 ---------------------------------
    Calculate Initial Gradient    
 ---------------------------------

 ion forces (au):
    1 C   (    0.00010   -0.00113   -0.00041 )
    2 H   (    0.00011    0.00037    0.00006 )
    3 H   (   -0.00017    0.00025    0.00019 )
    4 H   (   -0.00015    0.00033    0.00001 )
    5 H   (    0.00015   -0.00027   -0.00002 )
  C.O.M.  (    0.00007   -0.00081   -0.00030 )
|F|/nion  =    0.00028
max|Fatom|=    0.00121  (   0.062 eV/Angstrom)



@ Step             Energy     Delta E     Gmax     Grms     Xrms     Xmax   Walltime
@ ---- ------------------ ----------- -------- -------- -------- -------- ----------
@    0       -8.107103143  -8.107e+00  0.00121  0.00028  0.00008  0.00036        0.8        11
                                                     ok       ok       ok


 -----------------------------------------------------------------------------------
 ----------------------------- Optimization Step     1 -----------------------------
 -----------------------------------------------------------------------------------

 ---------------------------------
  Geometry for Step 1
 ---------------------------------

 == Bonds, Angles, and Torsions ==

 Number of Atoms = 5


 Units are Angstrom for bonds and degrees for angles

      Type          I     J     K     L     M      Value
      ----------- ----- ----- ----- ----- ----- ----------
    1 Stretch        C1    H2                      1.07760
    2 Stretch        C1    H3                      1.07769
    3 Stretch        C1    H4                      1.07782
    4 Stretch        C1    H5                      1.07805
    5 Bend           H2    C1    H3              109.31776
    6 Bend           H2    C1    H4              109.28680
    7 Bend           H2    C1    H5              109.68151
    8 Bend           H3    C1    H4              109.25891
    9 Bend           H3    C1    H5              109.65601
   10 Bend           H4    C1    H5              109.62445


 == XYZ format geometry ==

5

C        -0.000152     0.000432     0.000077
H         1.074377     0.076190    -0.029366
H        -0.290261    -0.637648     0.818680
H        -0.352468    -0.425823    -0.925064
H        -0.429665     0.980101     0.134089




 ---------------------------------
  Calculate Energy for Step 1
 ---------------------------------

     ================= Grassmann lmbfgs iteration =================
          >>> iteration started at Sat Jul  8 21:23:21 2023  <<<
     iter.                   Energy          DeltaE        DeltaRho
     --------------------------------------------------------------
        10      -8.107103546860e+00   -8.520274e-08    1.231941e-09
     *** tolerance ok. iteration terminated
          >>> iteration ended at   Sat Jul  8 21:23:21 2023  <<<

     =============  energy results (Molecule object)  =============


 number of electrons: spin up=     4.00000  down=     4.00000 (real space)


 total     energy    :   -8.1071035469e+00 (   -1.62142e+00 /ion)
 total orbital energy:   -3.3357959090e+00 (   -8.33949e-01 /electron)
 hartree energy      :    1.1896140303e+01 (    2.97404e+00 /electron)
 exc-corr energy     :   -3.1972634395e+00 (   -7.99316e-01 /electron)
 ion-ion energy      :    6.1835103027e+00 (    1.23670e+00 /ion)

 kinetic (planewave) :    6.3718476512e+00 (    1.59296e+00 /electron)
 V_local (planewave) :   -2.8603060567e+01 (   -7.15077e+00 /electron)
 V_nl    (planewave) :   -7.5827779750e-01 (   -1.89569e-01 /electron)
 V_Coul  (planewave) :    2.3792280607e+01 (    5.94807e+00 /electron)
 V_xc    (planewave) :   -4.1385858024e+00 (   -1.03465e+00 /electron)
 Viral Coefficient   :   -1.5235209772e+00

 orbital energy:
    -3.4796178e-01 (  -9.469eV)
    -3.4808144e-01 (  -9.472eV)
    -3.4879033e-01 (  -9.491eV)
    -6.2306442e-01 ( -16.955eV)

 == Center of Charge ==

 spin up    = (    0.0003    -0.0011    -0.0003 )
 spin down  = (    0.0003    -0.0011    -0.0003 )
      total = (    0.0003    -0.0011    -0.0003 )
 ionic      = (    0.0003    -0.0013    -0.0003 )

 == Molecular Dipole wrt Center of Mass ==

 mu   = (    0.0005    -0.0017    -0.0004 ) au
 |mu| =      0.0018 au (     0.0046 Debye )


 ---------------------------------
  Calculate Gradient for Step 1
 ---------------------------------

 ion forces (au):
    1 C   (    0.00008   -0.00009    0.00001 )
    2 H   (    0.00008    0.00042    0.00009 )
    3 H   (   -0.00017    0.00027    0.00026 )
    4 H   (   -0.00020    0.00030   -0.00014 )
    5 H   (    0.00023   -0.00037   -0.00001 )
  C.O.M.  (    0.00005   -0.00003    0.00002 )
|F|/nion  =    0.00017
max|Fatom|=    0.00043  (   0.022 eV/Angstrom)

      ----------------------
      Optimization converged
      ----------------------


  Step             Energy     Delta E     Gmax     Grms     Xrms     Xmax   Walltime
  ---- ------------------ ----------- -------- -------- -------- -------- ----------
@    1       -8.107103547  -4.041e-07  0.00043  0.00017  0.00005  0.00013        1.2        19
                                            ok       ok       ok       ok


 ---------------------------------
  Final Geometry 
 ---------------------------------

 == Bonds, Angles, and Torsions ==

 Number of Atoms = 5


 Units are Angstrom for bonds and degrees for angles

      Type          I     J     K     L     M      Value
      ----------- ----- ----- ----- ----- ----- ----------
    1 Stretch        C1    H2                      1.07760
    2 Stretch        C1    H3                      1.07770
    3 Stretch        C1    H4                      1.07784
    4 Stretch        C1    H5                      1.07800
    5 Bend           H2    C1    H3              109.32176
    6 Bend           H2    C1    H4              109.29112
    7 Bend           H2    C1    H5              109.67698
    8 Bend           H3    C1    H4              109.26346
    9 Bend           H3    C1    H5              109.65172
   10 Bend           H4    C1    H5              109.62049


 == XYZ format geometry ==

5

C        -0.000140     0.000418     0.000079
H         1.074390     0.076257    -0.029352
H        -0.290289    -0.637605     0.818722
H        -0.352501    -0.425776    -0.925086
H        -0.429629     0.980043     0.134087



 output psi to filename: ./ch4-pspw.movecs

 -----------------
 cputime in seconds
 prologue    : 1.923e-01
 main loop   : 9.739e-01
 epilogue    : 3.300e-02
 total       : 1.199e+00
 cputime/step: 5.126e-02 ( 19 evaluations, 5 linesearches)

 Time spent doing      total        step             percent
 total time            1.277905e+00 6.725813e-02     100.00%
 total FFT time        5.389055e-01 2.836345e-02      42.17%
 local potentials      3.664207e-03 1.928530e-04       0.29%
 non-local potentials  4.161042e-02 2.190022e-03       3.26%
 ffm_dgemm             1.471963e-02 7.747171e-04       1.15%
 fmf_dgemm             8.827582e-03 4.646096e-04       0.69%
 m_diagonalize         8.070900e-05 4.247842e-06       0.01%
 mmm_multiply          5.294000e-06 2.786316e-07       0.00%
 SCVtrans              1.262700e-05 6.645789e-07       0.00%

 >>> job completed at     Sat Jul  8 21:23:21 2023 <<<

Next rtdbstr={"constraints":null,"current_task":"task pspw optimize","dbname":"ch4-pspw","driver":null,"foundtask":false,"geometries":{"geometry":{"autosym":0,"autoz":0,"center":1,"charges":[6.0,1.0,1.0,1.0,1.0],"conv":1.88972598858,"coords":[-0.00026369498427201125,0.0007901067470504641,0.00014871390218976202,2.0303030982451484,0.14410396958633234,-0.05546779020207762,-0.5485660787579417,-1.2048990183475936,1.5471603156100777,-0.6661303302092375,-0.8046001592827439,-1.7481593262102584,-0.8118812316457504,1.8520128633335209,0.25338831361917336],"masses":[12.0,1.008,1.008,1.008,1.008],"nion":5,"symbols":["C","H","H","H","H"],"unita":[1.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,1.0],"velocities":[0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]}},"geometry":null,"nwinput_cur":38,"nwinput_lines":["Title \"CH4  test\"","","memory stack 500 mb heap 750 mb global 10 mb","","start ch4-pspw","","echo","","","","","","geometry noautosym noautoz","C        -0.000167     0.000612     0.000143","H         1.074360     0.076132    -0.029375","H        -0.290234    -0.637689     0.818651","H        -0.352445    -0.425876    -0.925066","H        -0.429688     0.980144     0.134092","end","","","nwpw","   simulation_cell","     SC 26.0","     ","   end","   ","   lmbfgs","","   xc pbe","   cutoff 30.0","   2d-hcurve","end","","task pspw optimize","","",""],"nwinput_nlines":38,"nwpw":{"cutoff":[30.0,60.0],"initialize_wavefunction":null,"mapping":3,"minimizer":2,"simulation_cell":{"unita":[26.0,0.0,0.0,0.0,26.0,0.0,0.0,0.0,26.0]},"xc":"pbe"},"permanent_dir":".","psp_library_dir":"","pspw":{"eigenvalues":[-0.3479617754830673,-0.34808143762654997,-0.34879032598021953,-0.6230644153945729],"energies":[-8.107103546859731,-3.33579590896882,11.896140303418617,-3.197263439547637,6.183510302719201,6.371847651153951,-28.603060567108667,-0.7582777974951738,23.792280606837235,-4.138585802356141,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1.9609956116897586e+243,6.052870644272e-310,2.999393627791262e-241,7.299540793070011e+170,8.410288607e-315,6.983507489299546e-251,0.0,0.0,0.0,0.0,0.0,3.099806181337924e+169,5.88757384886297e-310,2.999393627791262e-241,5.2150196931743e-310,8.887795736585723e+252],"energy":-8.107103546859731},"scratch_dir":".","title":"CH4  test"}
Next task =0

writing rtdbjson = ./ch4-pspw.json
